docs/authentication.rst

```diff
@@ -74,89 +121,89 @@ is how you would protect the secret endpoint with basic authentication::
-`OAuth Client Credential Grant Flow`.  The specification says that we should
-do this at the ``/oauth/token`` endpoint.  Our library handles this endpoint,
-but you must enable it with the ``api`` option in your configuration::
+If the credentials are valid, you will get an access token response that looks
+like this::
 
-    app.use(stormpath.init(app, {
-      api: true
-    }));
-
-We can then use our api keys from the last example, to get an access token::
-
-    $ curl -v --user apiKeyId:apiKeySecret http://localhost:3000/oauth?grant_type=client_credentials
-    {"access_token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJEUExSSTVUTEVNMjFTQzNER0xHUjBJOFpYIiwiaXNzIjoiaHR0cHM6Ly9hcGkuc3Rvcm1wYXRoLmNvbS92MS9hcHBsaWNhdGlvbnMvNWpvQVVKdFZONHNkT3dUVVJEc0VDNSIsImlhdCI6MTQwNjY1OTkxMCwiZXhwIjoxNDA2NjYzNTEwLCJzY29wZSI6IiJ9.ypDMDMMCRCtDhWPMMc9l_Q-O-rj5LATalHYa3droYkY","token_type":"bearer","expires_in":3600}
+    {
+      "access_token": "eyJ0eXAiOiJKV1QiL...",
+      "token_type": "bearer",
+      "expires_in": 3600
+    }
 
 The response is a JSON object which contains:
 
 - ``access_token`` - Your OAuth access token.  This can be used to authenticate
-  via subsequent requests.
-- ``token_type`` - This will always be ``'bearer'``.
+  on future requests.
+- ``token_type`` - This will always be ``"bearer"``.
 - ``expires_in`` - This is the amount of seconds (*as an integer*) for which
   this token is valid.
 
-With that token, we can make a request of our secret endpoint and supply the
-token as the bearer header in the request::
+With that token you can now make requests of your API.  This request is simpler,
+the only thing you need to do is supply the token in the ``Authorization`` header
+as a bearer token.  If you are using curl, that request looks like this::
 
-    $ curl -v -H "Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJEUExSSTVUTEVNMjFTQzNER0xHUjBJOFpYIiwiaXNzIjoiaHR0cHM6Ly9hcGkuc3Rvcm1wYXRoLmNvbS92MS9hcHBsaWNhdGlvbnMvNWpvQVVKdFZONHNkT3dUVVJEc0VDNSIsImlhdCI6MTQwNjY1OTkxMCwiZXhwIjoxNDA2NjYzNTEwLCJzY29wZSI6IiJ9.ypDMDMMCRCtDhWPMMc9l_Q-O-rj5LATalHYa3droYkY" http://localhost:3000/secret
-    {"message":"Hello, Robert"}
+    $ curl -v -H "Authorization: Bearer eyJ0eXAiOiJKV1QiL..." http://localhost:3000/secret
 
+In order to protect your API endpoint and allow this form of authenetication,
+you need to use the ``apiAuthenticationRequired`` middleware::
 
-Configuring OAuth Token Settings
---------------------------------
+    app.get('/secret', stormpath.apiAuthenticationRequired, function (req, res) {
+      res.json({
+        message: "Hello, " + req.user.fullname
+      });
+    });
 
-You can control the expiration time of the tokens that are created for the
-client credentials workflow.
+By default the access tokens are valid for one hour.  If you want to change
+the expiration of these tokens you will need to configure it in the server
+configuration, like this::
```

Comment:
Only for the password-based flow. The client-credentials flows is still done in our SDK.  We haven't implemented client credentials in the core API yet.